rediscluster/client.py

```diff
@@ -154,7 +151,7 @@ def __repr__(self):
     def pubsub(self, **kwargs):
         return ClusterPubSub(self.connection_pool, **kwargs)
 
-    def pipeline(self, transaction=None, shard_hint=None, use_threads=None):
```

Comment:
This change would require a note in the docs/Upgrading.md file stating that this argument can be removed from the implementers code.